src/pointfree/index.spec.js

```diff
@@ -42,6 +42,7 @@ const sequence = require('./sequence')
 const snd = require('../Pair/snd')
 const swap = require('./swap')
 const tail = require('./tail')
+const toPromise = require('./toPromise')
```

Comment:
This will need to point to `'../Async/toPromise.js'`, when it is moved over to the `Async` folder.